### ISSUE TITLE
Bad7zFile Error

### ISSUE THREAD

**Issue by PabloExperimental** [nekitmm/DLPacker#10]
Hi, 
I was trying to use this DLPacker on Google Colab, but after I run this code:
```
from dlpacker import DLPacker
dlp = DLPacker("/content/backbone.pdb")
dlp.reconstruct_protein(order="sequence", output_filename="structure_repacked.pdb")
```

I encounter this error:
```
Unzipping pretrained weight files...
---------------------------------------------------------------------------
Bad7zFile                                 Traceback (most recent call last)
[<ipython-input-34-b36edbf7401f>](https://localhost:8080/#) in <cell line: 1>()
----> 1 from dlpacker import DLPacker
      2 dlp = DLPacker("/content/backbone.pdb")
      3 dlp.reconstruct_protein(order="sequence", output_filename="structure_repacked.pdb")

5 frames
[/usr/local/lib/python3.10/dist-packages/dlpacker/__init__.py](https://localhost:8080/#) in <module>
----> 1 from dlpacker.dlpacker import DLPacker

[/usr/local/lib/python3.10/dist-packages/dlpacker/dlpacker.py](https://localhost:8080/#) in <module>
     55 
     56     print('Unzipping pretrained weight files...')
---> 57     unzip_weights(
     58         archive_path=f'{DEFAULT_WEIGHTS}.7z',
     59         output_dir=os.path.dirname(DEFAULT_WEIGHTS),

[/usr/local/lib/python3.10/dist-packages/dlpacker/utils.py](https://localhost:8080/#) in unzip_weights(archive_path, output_dir)
     75         print("Created single archive file: ", archive_path)
     76 
---> 77     with py7zr.SevenZipFile(archive_path, mode='r') as z:
     78             z.extractall(path=output_dir)
     79 

[/usr/local/lib/python3.10/dist-packages/py7zr/py7zr.py](https://localhost:8080/#) in __init__(self, file, mode, filters, dereference, password, header_encryption, blocksize, mp)
    407         except Exception as e:
    408             self._fpclose()
--> 409             raise e
    410         self._dict: Dict[str, IO[Any]] = {}
    411         self.dereference = dereference

[/usr/local/lib/python3.10/dist-packages/py7zr/py7zr.py](https://localhost:8080/#) in __init__(self, file, mode, filters, dereference, password, header_encryption, blocksize, mp)
    395         try:
    396             if mode == "r":
--> 397                 self._real_get_contents(password)
    398                 self.fp.seek(self.afterheader)  # seek into start of payload and prepare worker to extract
    399                 self.worker = Worker(self.files, self.afterheader, self.header, self.mp)

[/usr/local/lib/python3.10/dist-packages/py7zr/py7zr.py](https://localhost:8080/#) in _real_get_contents(self, password)
    427     def _real_get_contents(self, password) -> None:
    428         if not self._check_7zfile(self.fp):
--> 429             raise Bad7zFile("not a 7z file")
    430         self.sig_header = SignatureHeader.retrieve(self.fp)
    431         self.afterheader: int = self.fp.tell()

Bad7zFile: not a 7z file
```

Can you help me?

**Comment by nekitmm** [comment]
Were you able to solve this problem? I just published an update, which might solve this problem.

Just run 
`!pip install dlpacke`r
before you run the cell above

**Comment by nekitmm** [comment]
Yeah, I able to reproduce, will work on the solution. For now the problem can be overcome by running the following line in the colab:
`!7z e /usr/local/lib/python3.10/dist-packages/dlpacker/data/DLPacker_weights.7z.001`

**Comment by nekitmm** [comment]
Solved in #11 

Feel free to try again - the problem is gone, just make sure you install v1.0.1:
`pip install dlpacker>=1.0.1`
